README.md

```diff
@@ -1,12 +1,26 @@
-## RetroAchievements composer package
+[![N|Solid](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRAhk6glux07Qv-MJHXlBmK8LQ0gKALFPp2Yc_UTMGd9AMR_gRLKg)]()
+# RetroAchievements composer package
 
-Install with:
+RetroAchievements composer Package is a ......................................................
```

Comment:
For the description:
"RetroAchievements composer Package is a simple wrapper around the RetroAchievements.org API written in PHP"